docker-pg-databases/simulateur.sql

```diff
@@ -20,29 +20,21 @@ CREATE TABLE FireEvent (
     latitude FLOAT,
     longitude FLOAT,
     real_intensity INT,
+    sensor_id INT,
     start_date TIMESTAMP,
     end_date TIMESTAMP,
     is_real BOOLEAN,
```

Comment:
on laisse is real ou is verified du coup ?

Reply:
côté simu on sait si le feu est réel ou non